src/node-project.ts

```diff
@@ -1109,7 +1109,7 @@ export class NodeProject extends Project {
 
     const readDeps = (user: Record<string, string>, current: Record<string, string>) => {
```

Comment:
What's the use case of `current` being undefined?

At any rate, fix the type info:

```suggestion
    const readDeps = (user: Record<string, string>, current?: Record<string, string>) => {
```

Reply:
Made the change. Also, was never able to track down what is causing the issue but it's only on external projects as far as I can tell. I first noticed it before working on this and just trying to get projen to work with external packages in a different manner.

<img width="344" alt="Screen Shot 2020-10-21 at 9 31 54 AM" src="https://user-images.githubusercontent.com/2728868/96726913-7c795480-1380-11eb-8e37-567a9319eacb.png">